### PR TITLE
fix: conditional string equivalence

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Handles the case where the conditional in an `if` statement is a string.

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
Thanks for spotting this, @ShaunSHamilton.

File: python/py_helpers.test.py [hunk 1]
```diff
@@ -298,6 +298,17 @@ def test_whitespace_equivalence(self):
             Node(str_with_whitespace).is_equivalent(str_with_different_whitespace)
         )
 
+    def test_string_equivalence(self):
+        self.assertTrue(Node("'True'").is_equivalent('"""True"""'))
```
Comment (ShaunSHamilton) [inline python/py_helpers.test.py:302]: Is there a case where a developer might use `Node(code).is_equivalent...`? If so, this feels like more of a gotcha.

Comment (ojeytonwilliams) [reply python/py_helpers.test.py:302]: I'm not sure exactly how people will use this, but, `ast.parse('"""thing"""')` and `ast.parse("'thing'")` are the same up to column offsets. That makes it tricky to distinguish between the two.

Maybe this is confusing, but I suspect not _too_ confusing. After all `'thing' == "thing" == """thing"""` and `is_equivalent` has the same relationship.

Comment (ShaunSHamilton) [reply python/py_helpers.test.py:302]: Ah. I had actually forgotten Python had `"""`

**Review by ShaunSHamilton — DISMISSED** [review]
I wonder if we should wrap the return in one more layer of `unparse(parse())`. _just for just_

**Comment by ojeytonwilliams** [comment]
Can't be too careful.

**Pull request merged**